content/docs/tasks/telemetry/distributed-tracing/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Distributed Tracing
+description: This task shows you how Istio-enabled applications can be configured to collect trace spans.
```

Comment:
Avoid passive voice:
```suggestion
description: This task shows you how to configure Istio-enabled applications to collect trace spans.
```

Reply:
Have updated per @rcaballeromx suggestion, as it is still a task (i.e. configuring tracing). But if you have a suggestion for more appropriate wording I can update.